doc/introduction/templates.rst

```diff
@@ -189,6 +187,33 @@ of other templates.
 
         <div style='clear:both'></div>
 
+.. _intro_ref_temp_constr:
+
+Broadcasting function
+---------------------
+
+PennyLane offers a broadcasting function to easily construct templates. :func:`~.broadcast`
+takes other templates (or single quantum operations) and applies them to wires in a specific structure.
```

Comment:
```suggestion
takes other templates (or single quantum operations) and applies them to wires in a specific pattern.
```

Reply:
Or a comparable wording, this minor suggestion is mainly aimed at using the final chosen word from the signature of `broadcast`